clowdapp.yaml

```diff
@@ -32,6 +32,7 @@ objects:
     optionalDependencies:
       - notifications-backend
       - notifications-gw
+      - drift
```

Comment:
We have system-baseline as a dependency for hsp.  We should probably be consistent here and either make drift a dependency or make both optional dependencies